ppb/engine.py

```diff
@@ -70,25 +240,23 @@ def __init__(self, first_scene: Type, *,
            consequences. Consider passing via systems parameter instead.
         """
 
-        super(GameEngine, self).__init__()
+        super().__init__()  # FIXME: This is breaking the GameObject protocol
```

Comment:
O.o How is this breaking the protocol?

Reply:
Part of `GameObject` is that `__init__()` takes initial properties as kwargs.

```python
ge = GameEngine(spam='eggs')
ge.spam  # AttributeError
```

Reply:
Aha. I should play with run because we might be able to make that happen.

Reply:
It shouldn't be at the `run()` level? like, that's a convenience to make it easier to stand up an Engine.

Reply:
I mean, we convert the engine to only kwargs, run keeps its current API, we use run as the translation layer between.